docs/antora/modules/ROOT/pages/management/docker.adoc

```diff
@@ -24,38 +25,41 @@ In case a different node or network shall be used, xref:{url_config}[configure]
 
 == Stop
 
-.Inside of the `lisk-service` root folder
+.Inside the lisk-service root folder
 [source,bash]
 ----
 make down
 ----
 
 This will stop Lisk Service.
 
-== Show status of Docker containers
+== Show the status of Docker containers
 
 [source,bash]
 ----
 docker ps
 ----
 
-== Reset Lisk Service database
+== Reset the Lisk Service database
 
 To reset the database of Lisk Service, drop the respective MySQL and Redis databases.
 
 Before resetting the database, stop Lisk Service:
 
- make down
-
-Now connect to the Docker container in which you wish to reset the database. If you are not sure what to choose here, check the available containers via <<show-status-of-docker-containers>>.
-
 [source,bash]
 ----
-$ docker exec -it lisk-service_mysql_core_1 bash # <1>
+make down
```

Comment:
`make down` removes all the containers and the data. I could add a new `make stop` command instead for this.